CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
-# Unreleased
-- [...]
+#Unreleased
```

Comment:
Need a space between `#` and `Unreleased` or else it won't be displayed as a title (markdown stuff)